app/models/master_file.rb

```diff
@@ -723,6 +723,18 @@ def post_processing_file_management
     case Settings.master_file_management.strategy
     when 'delete'
       MasterFileManagementJobs::Delete.perform_now self.id
+    when 'move-ui-upload-only'
```

Comment:
also wondering in Ruby's convention, should we use "_" instead of "-"?

Reply:
Will change.